profiles/zaira.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/33151350/167124706-4332b930-2973-4486-aa00-9c9305ac64da.png",
+  "issueId": 66,
+  "name": "Eddie Jaoude",
+  "username": "eddiejaoude"
```

```suggestion
  "issueId": 1110,
  "name": "Zaira Hira",
  "username": "zairahira"
```
Forgot to change it to your credentials.